backend/ventserver/simulation/alarm_mute.py

```diff
@@ -10,20 +11,18 @@
 from ventserver.protocols.backend import states
 from ventserver.protocols.protobuf import mcu_pb
 
-def transform(
-    request: mcu_pb.AlarmMuteRequest,
-    response: mcu_pb.AlarmMute
-) -> None:
-    """Mute the current active alarm"""
-
-    response.active = request.active
-
 @attr.s
 class AlarmMuteService:
```

Comment:
It looks like the standard naming convention for service classes is `Service`/`Services`, because services are accessed through the python module name (it should be `alarm_mute_service = alarm_mute.Service()`) - see https://github.com/pez-globo/pufferfish-software/blob/1f7e1926a3dd0b4656c5efcc64e6fad36cf86b52/backend/ventserver/simulator.py#L123. Let's rename this class from `AlarmMuteService` to `Service`.